src/download/Cargo.toml

```diff
@@ -26,3 +26,6 @@ reqwest = { version = "0.9", optional = true }
 futures = "0.1"
 hyper = "0.12"
 tempdir = "0.3.4"
+tokio = "0.1.11"
+tokio-tls = "0.2.1"
+native-tls = "0.2.1"
```

Comment:
You've managed to miss the newline off this file, which while minor is super-irritating :D